certificate/certificate.go

```diff
@@ -141,9 +141,9 @@ func (cert *Certificate) serializeNetworkPart() []byte {
 }
 
 // SignNetworkPart signs network part in certificate
-func (cert *Certificate) SignNetworkPart(key crypto.PrivateKey) ([]byte, error) {
+func (cert *Certificate) SignNetworkPart(key crypto.PrivateKey, serialized []byte) ([]byte, error) {
```

Comment:
No need to pass serialized. We need to call serialize() inside SignNetworkPart method